src/routes/+page.svelte

```diff
@@ -0,0 +1,444 @@
+              content.
+            </p>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="image-wrapper">
+            <img src={streamHighlights} alt="Stream Highlights" />
+          </div>
+          <div class="content-wrapper">
+            <h3>Stream Highlights</h3>
+            <p>Easily create and share live stream highlights to engage and grow your audience.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="features-solutions">
+      <h2>Cost-Effective Solution</h2>
+      <p class="solution-description">
+        Significantly more affordable than current market options,<br />
+        making high-quality video streaming accessible to all.
+      </p>
+      <div class="comparison-grid">
+        <div class="solution-card">
+          <div class="solution-card-header main">
+            <h3 class="bold our-solution">Scuffle&apos;s</h3>
+            <h3 class="our-solution">Video Streaming Solution</h3>
+          </div>
+          <div class="row-wrapper">
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+          </div>
+        </div>
+        <div class="solution-card">
+          <div class="solution-card-header">
+            <h3>Other<br />Video Streaming Solutions</h3>
+          </div>
+          <div class="row-wrapper">
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+            <div class="row"></div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <EmailForm />
+  </div>
+</div>
+
+<style>
+  .hero-section-container {
+    background-color: var(--color-light100);
+    max-width: 64rem;
+    margin: 0 auto;
+    padding: 1rem;
+    width: 100%;
+
+    :global {
+      .hero-section {
+        h1 {
+          line-height: 1.2;
+        }
+      }
+
+      .hero-section-title {
+        font-weight: 400;
+      }
+    }
+
+    /* Will be changed later when picture is added */
+    .github-bubble {
+      background: white;
+      padding: 0.75rem 1rem;
+      border-radius: 1rem;
+      position: relative;
+    }
+
+    /* Will be changed later when picture is added */
+    .github-bubble::after {
+      content: '';
+      position: absolute;
+      right: -10px;
+      top: 50%;
+      transform: translateY(-50%);
+      border-left: 0.625rem solid white;
+      border-top: 0.625rem solid transparent;
+      border-bottom: 0.625rem solid transparent;
+    }
+  }
+
+  /* Will be changed later when picture is added */
+  .box {
+    width: 100px;
+    height: 100px;
+    background: var(--color-light200);
+    border-radius: 0.5rem;
+  }
```

Comment:
`px` im not apposed to leaving this since the comment says you will change it but; i suspect we might forget to do that

Reply:
i won't forget ill remove this whole box